pkg/cmd/resource_resolver.go

```diff
@@ -143,6 +143,9 @@ func (rv *resourceResolver) isVerbSupportedBy(verb string, resource apismeta.API
 	if verb == rbac.VerbAll {
 		return true
 	}
+	if resource.Name == "podsecuritypolicies" && verb == "use" {
```

Comment:
nit: If we support any other special verbs in the future, we can write this like the following.
```
    specials = []schema.XXX{        
          {
              Group:    "extensions",
              Resource: "podsecuritypolicies",
              Verb: "use",
          },
      }
```

```
for _, s := range specials {     
    if resource.Name ==  s.Resource && verb == s.Verb {
        return true
    }
}
```